app/templates/components/favorite-article-button.hbs

```diff
@@ -0,0 +1,3 @@
+<button class="btn btn-sm {{@class}}" {{action @onClick}} data-test-favorite-article-button>
```

Comment:
Two things :)
First is that we shouldn't be using `@prop` as this is an Octane feature. Right? @mansona Any ideas?
Second I prefer not to override the name of something like `class` with `@class`. Usually in my apps I name it to something like `wrappingClass`.
What do you think?

Reply:
According to [Ember 3.1 and 3.2 Beta Released](https://blog.emberjs.com/2018/04/13/ember-3-1-released.html) blog post, named arguments, aka `@prop`, was introduced in Ember@3.1.

Can you help me understand how `class` is being overridden? `class` is a property set on the `favorite-article-button` and it's being treated as a property that is being passed in - thus the usage of `{{@class}` to identify it as such.

Reply:
As we discussed, I will remove usages of named args and add back in tagNames.

The reason as I understand it is that named args was made for Octane and empty tagNames string usage is not in the guides.